server/base/src/main/java/org/apache/accumulo/server/util/SystemPropUtil.java

```diff
@@ -89,8 +89,10 @@ private static String validateSystemProperty(ServerContext context, SystemPropKe
       log.trace("Encountered error setting zookeeper property", iae);
       throw iae;
     }
-    if (Property.isValidTablePropertyKey(property)) {
-      PropUtil.validateProperties(context, key, Map.of(property, value));
+    if (property.startsWith(Property.TABLE_PREFIX.getKey())) {
+      throw new IllegalArgumentException(
+          "Table property " + property + " cannot be set at the system level."
+              + " Set table properties at the namespace or table level.");
```

Comment:
Would be nice to somehow consolidate this exception message between the two places. Maybe could move this into a method in PropUtil, like PropUtil.checkForTablePropAtSystemLevel(property).  That method does the prefix check and throws the exception and both places can call it.

Reply:
Fixed this in 220b656